pkg/image/builder.go

```diff
@@ -101,6 +101,13 @@ func GetPrivateImages(upstreamDir string, checkedImages map[string]ImageInfo, al
 			if err != nil {
 				return err
 			}
+
+			// Skip charts folders containing subcharts.  The kustomization for subcharts will be defined
+			// inside of those charts, and should not be referenced at the parent chart level.
+			if info.IsDir() && info.Name() == "charts" {
+				return filepath.SkipDir
+			}
+
```

Comment:
can you please add a testcase for subcharts.

Reply:
I will add one to the next PR, we ran into a separate bug with sub-sub-charts.  If I add it to tests now the tests will fail.